files/image_config/fstrim/fstrim.service

```diff
@@ -3,4 +3,4 @@ Description=Discard unused blocks
 
 [Service]
 Type=oneshot
-ExecStart=/sbin/fstrim -av
+ExecStart=/usr/bin/sonic-fstrim
```

Comment:
can we run sonic-fstrim when it's available?

Reply:
Yes, we can. I have changed the permissions in the code base itself and built the image and tested it.

Reply:
@rkdevi27 can we make platform_log_ssd_health more generically and run it from fstrim.service? And then we don't need sonic-fstrim any more.

Reply:
Sujin, 
As suggested, I have changed log_ssd_health as generic.